yellow-paper/docs/private-message-delivery/private-msg-delivery.md

```diff
@@ -19,20 +19,53 @@ If Alice executes a function that generates a note for Bob:
 
 ## Constraining Message Delivery
 
-The protocol will enable app developers to constrain the correctness of the following:
+The protocol will enable account contract developers to constrain the correctness of the following:
```

Comment:
Not sure I agree with this change. It's the app who decides whether to constrain note encryption and tagging. The user just signals their preferred method via the registry, but it's the app who can decide to follow it or not, and whether to constrain it or not. Account contract devs do not deal with message delivery (wallet clients devs do, though).

Reply:
Yes, you're right.